gamecenter/logger.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from collections import OrderedDict
+import json
+import logging
+
+import arrow
+from flask import request, g
+
+
+class RankTimedRotatingFileHandler(logging.handlers.TimedRotatingFileHandler):
+    """
+    Handles our log with care so the poor child knows where to go
+
+    It will change what is appended to the filename based on the time it's rotated.
+    Suffixes will appear in the form %Y-%m-%d_%H-%M-%S E.g.:
+    /var/log/rank/log-2015-11-04_00-00-00
+
+    Every midnight, the logs rotate to a new file
```

Comment:
One thing we maybe? need to think about is if we do a redeploy at noon, do we lose 12 hours of logs?